### PR TITLE
Remove coreapi requirement

### LINKED ISSUE #1969: Remove coreapi requirement
## Description

The ``coreapi`` requirement should be removed from ``requirements/base.txt``.

## Rationale

From the [CoreAPI](https://github.com/core-api/core-api/) page:

>DEPRECATION NOTICE: As of djangorestframework 3.9, the plan is to phase out CoreAPI in favor of OpenAPI as the default schema representation. New projects should consider using OpenAPI rather than CoreAPI. See the DRF 3.9 release announcment for more details.

The [DRF 3.9 page](https://www.django-rest-framework.org/community/3.9-announcement/#whats-next) has an example of how to use the integrated OpenAPI support.

## Use case(s) / visualization(s)

None. It's a requirement for a module no longer developed. If it's used somewhere in the template that I've missed, it should be replaced with [OpenAPI](https://www.openapis.org/) equivalents.

I only noticed it because I'm going through a Django project generated by this cookiecutter and can't explain why the requirement exists.

### PR DESCRIPTION
**Pull request by cole**
##  Description

Removes coreapi from requirements, per https://github.com/pydanny/cookiecutter-django/issues/1969

## Rationale

Coreapi is deprecated and archived, and not appropriate for new projects.

Fixes #1969

### CODE REVIEW

**Comment by morenoh149** [comment]
closes #1969

File: CONTRIBUTORS.rst [hunk 1]
```diff
@@ -87,6 +87,7 @@ Listed in alphabetical order.
   Chris Pappalardo         `@ChrisPappalardo`_
   Christopher Clarke       `@chrisdev`_
   Cole Mackenzie           `@cmackenzie1`_
+  Cole Maclean             `@cole`_                      @cole
```
Comment (browniebroke) [inline CONTRIBUTORS.rst]: Just a note that this is a broken link in rst. To fix it, you should add the target as the others further down the file.

Comment (cole) [reply CONTRIBUTORS.rst]: Oops, of course. Thanks!

**Comment by browniebroke** [comment]
Thanks 🎉

**Pull request merged**